DelHel/CDelHel.cpp

```diff
@@ -353,6 +381,79 @@ void CDelHel::SaveSettings()
 	this->SaveDataToSettings(PLUGIN_NAME, "DelHel settings", ss.str().c_str());
 }
 
+void CDelHel::ReadCustomConfigs()

```

Comment:
I would propose not requiring a `customconfigs.json` to exist, in case no custom configs are in use.
While the plugin would load fine, you'd get a error message on every startup and would have to create a dummy/empty config just to silence that... Could you add a check to see if the config exists, and skips loading if it doesn't (maybe logging a **debug** log in that case)?